#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_changed_api.patch.dpatch by  <vdr04 -at- wopr.deltab.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: 01_changed_api

@DPATCH@
--- ./ffm.cpp	2006-11-10 16:55:07.000000000 +0100
+++ ./ffm.cpp	2006-11-10 17:54:17.000000000 +0100
@@ -1817,7 +1817,7 @@
 
     /* check filename in case of an image number is expected */
     if (oc->oformat->flags & AVFMT_NEEDNUMBER) {
-        if (av_filename_number_test(oc->filename) < 0) {
+        if (filename_number_test(oc->filename) < 0) {
             print_error(oc->filename, AVERROR_NUMEXPECTED);
             return false;
         }
